### ISSUE TITLE
VNC Server

### ISSUE THREAD

**Issue by majo99** [filebot/filebot#57]
Hi I am very new to all this so please bare with me. I have installed version 4.8.2 on a server running linux ubuntu 18.04. I installed it via atomik toolkit and want to use it with tightvnc. The problem I am having is that when I go to open it in vnc nothing happens. I have just spent 3 hours searching for links between filebot and vnc and online resources linking the two subjects are scarce to say the least. Any advice or help would be greatly appreciated.
Thanks

**Comment by rednoah** [comment]
If you run `filebot` in the console, what does the console output say?
https://www.filebot.net/forums/viewtopic.php?f=5&t=1868

**Comment by majo99** [comment]
![image](https://user-images.githubusercontent.com/46468353/50812567-2c481400-130b-11e9-81d6-ab9756035344.png)
![image](https://user-images.githubusercontent.com/46468353/50812591-44b82e80-130b-11e9-8222-33df79892165.png)
 As I mentioned I am a complete novice but I really cant figure out what i am doing wrong. Probably something basic but after hours of searching I have no solution. I may even be missing a run command in terminal but i have not been able to find any

**Comment by majo99** [comment]
![image](https://user-images.githubusercontent.com/46468353/50813156-65818380-130d-11e9-8cee-75147f1c36e1.png)
![image](https://user-images.githubusercontent.com/46468353/50813205-919d0480-130d-11e9-8969-a492b993e94c.png)
![image](https://user-images.githubusercontent.com/46468353/50813224-9feb2080-130d-11e9-961d-375be471e09b.png)
![image](https://user-images.githubusercontent.com/46468353/50813248-b2655a00-130d-11e9-96ce-ad6ac7ba5447.png)
![image](https://user-images.githubusercontent.com/46468353/50813270-c315d000-130d-11e9-810f-29196d40d022.png)
![image](https://user-images.githubusercontent.com/46468353/50813295-d4f77300-130d-11e9-81be-041c75608744.png)
![image](https://user-images.githubusercontent.com/46468353/50813311-e0e33500-130d-11e9-89a8-f82ec121c1ca.png)
![image](https://user-images.githubusercontent.com/46468353/50813337-f0627e00-130d-11e9-9665-d1b15d725fa1.png)
![image](https://user-images.githubusercontent.com/46468353/50813359-fd7f6d00-130d-11e9-93e8-a838db52dfdf.png)

**Comment by majo99** [comment]
Please accept my apologies I have uninstalled the snap and installed the deb version and all is working fine now

**Comment by majo99** [comment]
Another noob question however is how do i activate the licence key ? I have read the forum and the part about the *psm and I have received the attachment which opened up in notedpad, copied and pasted using the instructions but getting nowhere. In honesty I don't the instructions are set up with novices in mind so any more help would be appreciated

**Comment by rednoah** [comment]
You can find instructions and screenshots on how to activate license keys here:
https://www.filebot.net/forums/viewtopic.php?f=8&t=6121

TL;DR Don't worry about activating your license at all. FileBot will work just fine without, and ask for it if and when it is required, and the steps will be obvious when that happens. ;)
